exercises/parallel-letter-frequency/description.md

```diff
@@ -4,4 +4,4 @@ Count the frequency of letters in texts using parallel computation.
 
 Parallelism is about doing things in parallel that can also be done sequentially.
 A common example is counting the frequency of letters.
-Create a function that returns the total frequency of each letter in a list of texts and that employs parallelism.
+Employ parallelism to return the total frequency of each letter in a list of texts.
```

Comment:
```suggestion
Employ parallelism to calculate the total frequency of each letter in a list of texts.
```

Not because I dislike 'return', but because I feel 'calculate' fits better.

Reply:
There is an ongoing discussion about this exercise and whether solutions must use parallelism vs concurrency vs allowing non interleaved solutions. It might be worth holding off on any changes here for a bit to reduce churn. On the other hand, this said parallelism before and that word isn't changing.

Reply:
@IsaacG, I made a call to keep the change with @MatthijsBlom's `s/return/calculate/` suggestion since -- as you say -- the word parallelism is there as it was before (and still in the exercise name/slug 😅) but LMK if you feel strongly that it should be removed from this PR. 

My feeling is that this way, the function-specific language issue is resolved, whereas it may be forgotten/overlooked amid the parallelism/concurrency debate? 

Thanks for your time and your feedback 🙇